<ui version="4.0" >
 <class>AIMUserInfoWidget</class>
 <widget class="QWidget" name="AIMUserInfoWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>475</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>360</width>
    <height>400</height>
   </size>
  </property>
  <layout class="QGridLayout" >
   <property name="leftMargin" >
    <number>0</number>
   </property>
   <property name="topMargin" >
    <number>0</number>
   </property>
   <property name="rightMargin" >
    <number>0</number>
   </property>
   <property name="bottomMargin" >
    <number>0</number>
   </property>
   <item row="0" column="0" >
    <widget class="QLabel" name="lblNickName" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Maximum" hsizetype="Maximum" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string>Nickname:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QLineEdit" name="txtNickName" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="0" column="2" >
    <widget class="QLabel" name="lblScreenName" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Maximum" hsizetype="Maximum" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string>Screen name:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3" >
    <widget class="QLineEdit" name="txtScreenName" >
     <property name="readOnly" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="lblWarnLevel" >
     <property name="text" >
      <string>Warning level:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="QLineEdit" name="txtWarnLevel" >
     <property name="readOnly" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="2" >
    <widget class="QLabel" name="lblIdleTime" >
     <property name="text" >
      <string>Idle minutes:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3" >
    <widget class="QLineEdit" name="txtIdleTime" >
     <property name="readOnly" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>50</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="2" >
    <widget class="QLabel" name="lblOnlineSince" >
     <property name="text" >
      <string>Online since:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3" >
    <widget class="QLineEdit" name="txtOnlineSince" >
     <property name="readOnly" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="4" >
    <widget class="QLabel" name="lblAwayMessage" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Minimum" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string>Away message:</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignTop</set>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="4" >
    <widget class="QTextBrowser" native="1" name="txtAwayMessage" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="4" >
    <widget class="QLabel" name="textLabel1" >
     <property name="text" >
      <string>Profile:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="4" >
    <widget class="QFrame" native="1" name="userInfoFrame" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize" >
      <size>
       <width>64</width>
       <height>16</height>
      </size>
     </property>
     <property name="lineWidth" stdset="0" >
      <number>0</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>txtNickName</tabstop>
  <tabstop>txtScreenName</tabstop>
  <tabstop>txtWarnLevel</tabstop>
  <tabstop>txtIdleTime</tabstop>
  <tabstop>txtOnlineSince</tabstop>
  <tabstop>txtAwayMessage</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
